src/openrct2-ui/windows/EditorObjectSelection.cpp

```diff
@@ -125,27 +126,47 @@ struct ObjectPageDesc
     { STR_OBJECT_SELECTION_WALLS_FENCES,                SPR_TAB_SCENERY_WALLS,      true  },
-    { STR_OBJECT_SELECTION_PATH_SIGNS,                  SPR_TAB_SCENERY_SIGNAGE,    true  },
-    { STR_OBJECT_SELECTION_FOOTPATHS,                   SPR_G2_LEGACY_PATH_TAB,      true  },
+    { STR_OBJECT_SELECTION_FOOTPATH_SURFACES,           SPR_G2_PATH_SURFACE_TAB,    false },
+    { STR_OBJECT_SELECTION_FOOTPATH_RAILINGS,           SPR_G2_PATH_RAILINGS_TAB,   false },
+    { STR_OBJECT_SELECTION_FOOTPATHS,                   SPR_G2_LEGACY_PATH_TAB,     true  },
     { STR_OBJECT_SELECTION_PATH_EXTRAS,                 SPR_TAB_SCENERY_PATH_ITEMS, false },
-    { STR_OBJECT_SELECTION_SCENERY_GROUPS,              SPR_TAB_SCENERY_STATUES,    false },
-    { STR_OBJECT_SELECTION_PARK_ENTRANCE,               SPR_TAB_PARK,               false },
+    { STR_OBJECT_SELECTION_PATH_SIGNS,                  SPR_TAB_SCENERY_SIGNAGE,    true  },
     { STR_OBJECT_SELECTION_WATER,                       SPR_TAB_WATER,              false },
-
+    { STR_OBJECT_SELECTION_TERRAIN_SURFACES,            SPR_G2_TAB_LAND,            true  },
+    { STR_OBJECT_SELECTION_TERRAIN_EDGES,               SPR_G2_TERRAIN_EDGE_TAB,    true  },
+    { STR_OBJECT_SELECTION_PARK_ENTRANCE,               SPR_TAB_PARK,               false },
+    { STR_OBJECT_SELECTION_STATIONS,                    SPR_G2_RIDE_STATION_TAB,    true  },
+    { STR_OBJECT_SELECTION_MUSIC,                       SPR_TAB_MUSIC_0,            false },
     // Dummy place holder for string objects
     { STR_NONE,                   static_cast<uint32_t>(SPR_NONE),                  false },
+};
 
-    { STR_OBJECT_SELECTION_TERRAIN_SURFACES,            SPR_G2_TAB_LAND,            true  },
-    { STR_OBJECT_SELECTION_TERRAIN_EDGES,               SPR_G2_TERRAIN_EDGE_TAB,            true  },
-    { STR_OBJECT_SELECTION_STATIONS,                    SPR_G2_RIDE_STATION_TAB,               true  },
-    { STR_OBJECT_SELECTION_MUSIC,                       SPR_TAB_MUSIC_0,            false },
-    { STR_OBJECT_SELECTION_FOOTPATH_SURFACES,           SPR_G2_PATH_SURFACE_TAB,      false },
-    { STR_OBJECT_SELECTION_FOOTPATH_RAILINGS,           SPR_G2_PATH_RAILINGS_TAB,   false },
+// Order of which the contents of each tab is displayed.
+static const std::map<int16_t, ObjectType> TabOrderMap = {
+        {0, ObjectType::Ride},
+        {1, ObjectType::SceneryGroup},
+        {2, ObjectType::SmallScenery},
+        {3, ObjectType::LargeScenery},
+        {4, ObjectType::Walls},
+        {5, ObjectType::FootpathSurface},
+        {6, ObjectType::FootpathRailings},
+        {7, ObjectType::Paths},
+        {8, ObjectType::PathBits},
+        {9, ObjectType::Banners},
+		{10, ObjectType::Water},
+        {11, ObjectType::TerrainSurface},
+        {12, ObjectType::TerrainEdge},
+        {13, ObjectType::ParkEntrance},
+        {14, ObjectType::Station},
+        {15, ObjectType::Music},
+        {16, ObjectType::ScenarioText}
 };
```

Comment:
This seems like it could be an array

Reply:
How would I go about doing that? 

```
int TabOrderMap(ObjectType) {
       int x = 17;
       int16_t arr[x] = {ObjectType::Ride,
                         ObjectType::SceneryGroup,
                         ObjectType::SmallScenery,
                         ObjectType::LargeScenery,
                         ObjectType::Walls,
                         ObjectType::FootpathSurface,
                         ObjectType::FootpathRailings,
                         ObjectType::Paths,
                         ObjectType::PathBits,
                         ObjectType::Banners,
                         ObjectType::Water,
                         ObjectType::TerrainSurface,
                         ObjectType::TerrainEdge,
                         ObjectType::ParkEntrance,
                         ObjectType::Station,
                         ObjectType::Music,
                         ObjectType::ScenarioText};
}``` 

Like so?

Reply:
Done :)